src/App.jsx

```diff
@@ -7,7 +7,9 @@ function App() {
       <header className="App-header">
         <img src={logo} className="App-logo" alt="logo" />
         <p>
```

Comment:
- [ ] You did an amazing job to to setup the project, but there is a small issue for your project to run properly without any error, You have deleted` logo.svg` and `App.css file`, but you are stilling calling them on the top of your` App.jsx` file. this is causing the error because those files are not there. please delete line one and two and make sure that you not using components which are not there, for your application to run successfully.